tests/regression/nextcloud-rule-exclusions-plugin/9508551.yaml

```diff
@@ -23,3 +23,37 @@ tests:
             version: HTTP/1.1
           output:
             no_log_contains: id "931130"
+  - test_title: 9508552-2
```

Comment:
Aren't these two tests identical?

Reply:
Looks like the duplicate tests were already there beforehand, it's fixed now.